libs/angular/src/shared/components/password-strength/password-strength.component.ts

```diff
@@ -98,32 +98,13 @@ export class PasswordStrengthComponent implements OnChanges {
 
     const strengthResult = this.passwordGenerationService.passwordStrength(
       masterPassword,
-      this.getPasswordStrengthUserInput()
+      [...(this.name?.trim().toLowerCase().split(" ") ?? [])],
```

Comment:
This unpacking looks unnecessary, you're just wrapping it in a new array anyway. `split` returns an array, so can you do this?

```suggestion
      this.name?.trim().toLowerCase().split(" ") ?? [],
```

Also, does `split` ever return null? do you need the null coalescing?

Reply:
Ah, yep, that's unnecessary unpacking. 

I wasn't worried about `split` returning null, but the `this.name?` could have been. However, looking again, we can just pass in null for `userInputs` anyway as `passwordStrength` safely handles the null case.